src/components/CityWeatherCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './CityWeatherCard.module.css';
+
+export interface CityWeatherCardProps {
+    name: string;
+    temperature: string;
+    conditions: string;
+    wind_speed: string;
+    humidity: string;
+}
+
+const CityWeatherCard: React.FC<CityWeatherCardProps> = ({
```

Comment:
This is 100% valid but one small note from me but I often find it easier to just type the params & return type rather than setting the type because it fits into standard TS functions more closely.

E.G
```ts
const CityWeatherCard = ({ ... }: CityWeatherCardProps): React.ReactNode => { ... }
```

With that said that might not always be an option based on the `tsconfig` settings of the project & also this is just my personal preference & what you already have looks great